pkg/scheduler/scheduler.go

```diff
@@ -408,7 +383,7 @@ func (sched *Scheduler) recordSchedulingFailure(podInfo *framework.PodInfo, err
 // If it succeeds, it adds the name of the node where preemption has happened to the pod spec.
 // It returns the node name and an error if any.
 func (sched *Scheduler) preempt(ctx context.Context, state *framework.CycleState, fwk framework.Framework, preemptor *v1.Pod, scheduleErr error) (string, error) {
```

Comment:
It looks like we don't have any unit tests covering sched.preempt?

Could you add some? We have some in `test/integration/preemption_test.go`, but maybe some of those should be unit instead.